src/libraries/Microsoft.PowerFx.Interpreter/UDF/UserDefinedTexlFunction.cs

```diff
@@ -38,6 +38,17 @@ public async Task<FormulaValue> InvokeAsync(FormulaValue[] args, CancellationTok
             return result;
         }
 
+        public async Task<FormulaValue> InvokeAsync(FormulaValue[] args, CancellationToken cancel, StackDepthCounter stackMarker, ReadOnlySymbolValues symbols)
```

Comment:
>InvokeAsync [](http://example.com/codeflow?start=40&length=11)

Let's deal with UDFs separately. We have to sort out how to plumb these through.